ProjFS.Mac/PrjFSKext/PrjFSKext/KauthHandler.cpp

```diff
@@ -723,6 +726,17 @@ static bool ShouldHandleVnodeOpEvent(
 
     *kauthResult = KAUTH_RESULT_DEFER;
     
+    if (ActionBitIsSet(action, KAUTH_VNODE_ACCESS))
+    {
+      // From kauth.h:
+      //    "The KAUTH_VNODE_ACCESS bit is passed to the callback if the authorisation
+      //    request in progress is advisory, rather than authoritative.  Listeners
+      //    performing consequential work (i.e. not strictly checking authorisation)
+      //    may test this flag to avoid performing unnecessary work."
+      
+        return false;
```

Comment:
I noticed each other place returning false in this function was explicitly setting `*kauthResult = KAUTH_RESULT_DEFER`. Should this instance be consistent?

Reply:
Good idea, I agree it's better to be consistent